### PR TITLE
Point cloud update

### PR DESCRIPTION
**Pull request by liealex**

### CODE REVIEW

File: examples/point_cloud/main.cpp [hunk 1]
```diff
@@ -15,8 +15,27 @@ static char const* p_xyz_frame_output_file_name = "XYZFrame.bin";
 static const char kUsage[] =
 R"(PointCloud.
     kUsage:
-      PointCloud --input_file=<Input depth file> --ccb_file=<ccb_file> --mode=<mode> --output_folder=<Output_folder>
-      PointCloud (-h | --help) 
+      PointCloud (-h | --help)
+      first-frame [-input_file | --input_file <Input depth file>] [-ccb_file | --ccb_file <ccb_file>] [-mode | --mode <mode>] [-output_folder | --output_folder <Output_folder>]
```
Comment (dNechita) [inline examples/point_cloud/main.cpp:19]: "first-frame" should not be written in this menu

File: examples/point_cloud/main.cpp [hunk 2]
```diff
@@ -15,8 +15,27 @@ static char const* p_xyz_frame_output_file_name = "XYZFrame.bin";
...
+
+    Options:
+      -h --help                         Show this screen.
+      --input_file <Input depth file>   Path to the input file (.bin)
+      --ccb_file <ccb_file>             Path to the ccb content
+      --mode <mode>                     Mode to capture data in. [default: 0]
+      --output_folder <Output_folder>   Path to store the output file
+
+    Note: --m argument supports both index and string (0/sr-native)
```
Comment (dNechita) [inline examples/point_cloud/main.cpp:28]: "--m" that is written here hasn't been mentioned before. Only -mode |--mode has been mentioned. This inconsistency can create confusion.

File: examples/point_cloud/main.cpp (hunk 1 above)
Comment (liealex) [reply examples/point_cloud/main.cpp]: Solved

File: examples/point_cloud/main.cpp (hunk 2 above)
Comment (liealex) [reply examples/point_cloud/main.cpp]: Solved

**Review by dNechita — APPROVED** [review]

**Pull request merged**